documentation/book/proc-deploying-cluster-operator-kubernetes-to-watch-whole-cluster.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// assembly-cluster-operator.adoc
+// assembly-operators-cluster-operator.adoc
+
+[id='deploying-cluster-operator-kubernetes-to-watch-whole-cluster-{context}']
+= Deploying the Cluster Operator to watch the whole cluster
+
+Cluster Operator can be installed to manage Kafka, Kafka Connect and Kafka Mirror Maker in the whole {ProductPlatformName} cluster across all {Namespaces}.
+In this mode, the Cluster Operator will automatically start managing clusters in any new {Namespaces} which will be created while it is running and drop any {Namespaces} which will be deleted.
```

Comment:
I know what you mean about the deletion part, but it doesn't read right and I can't think of a wording which would explain it without using a lot of words, so on balance I think we can just get rid of that part.

```suggestion
In this mode, the Cluster Operator will automatically start managing clusters in any new {Namespaces} which will be created while it is running.
```